### PR TITLE
Ignore Internal Operation Tables From Vdiff

### PR DESCRIPTION
**Pull request by makinje16**
Signed-off-by: Malcolm Akinje <makinje@slack-corp.com>

## Description

This PR modifies VDiff by now ignoring tables that pass the `IsInternalOperationTableName` method.
## Checklist
- [x] Tests were added or are not required
- [ ] Documentation was added or is not required

### CODE REVIEW

File: go/vt/wrangler/vdiff.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ import (
 	"vitess.io/vitess/go/vt/key"
 	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/logutil"
+	vtschema "vitess.io/vitess/go/vt/schema"
```
Comment (ajm188) [inline go/vt/wrangler/vdiff.go:40]: I don't think we should rename this import. I read it as `vschema` several times and kept wondering why IsInteralOperationTableName` was in that package.

If there's a local variable in this file preventing us from using this name, I think we should rename that variable.

**Review by ajm188 — COMMENTED** [review]
very excited for this!

File: go/vt/wrangler/vdiff.go (hunk 1 above)
Comment (makinje16) [reply go/vt/wrangler/vdiff.go]: unrenamed and should be in next commit

**Review by rohit-nayak-ps — APPROVED** [review]
lgtm

**Pull request merged**